### DISCUSSION TITLE
How to generate a nice looking HTML report?

### DISCUSSION THREAD

**Discussion by mmahut** [stepci/stepci#97] [Q&A]
Is there a way to generate a nice looking HTML such as on the playground on the main page?

**Comment by mishushakov** [comment]
Thanks for the idea!
How important is this feature for you and what use-case do you see for that?

Let me know 😉

**Reply by mmahut** [reply]
Not a show stopper, just something that would be super nice to have.

We are mostly interested about the network transfer breakdown graph.

**Reply by mishushakov** [reply]
Great. I've added it to #64

**Comment by sebastianwis** [comment]
Hey Marek @mmahut  
Thank you once again for your contribution here! 
It would be amazing to welcome you to our Discord if you haven't joined it yet: 
https://discord.com/invite/KqJJzJ3BTu
This is where we keep users up to date with updates, features and bug resolutions.
